views/install.sh.erb

```diff
@@ -112,8 +112,6 @@ elif test -f "/etc/redhat-release"; then
 
   # If /etc/redhat-release exists, we act like RHEL by default
   if test "$platform" = "fedora"; then
-    # FIXME: stop remapping fedora to el
-    # FIXME: remove client side platform_version mangling and hard coded yolo
```

Comment:
nope those still need to get fixed and moved to platform.rb and out of the install.sh code